deploy-tool/main.go

```diff
@@ -78,6 +79,7 @@ type TerraformOutput struct {
 	SpecificManifests struct {
 		Value map[string]struct {
 			KubernetesNamespace string           `json:"kubernetes-namespace"`
+			CertificateFQDN     string           `json:"certificate_fqdn"`
```

Comment:
Prefer using kebab-case everywhere, not cause it's necessarily better than snake_case but just for consistency.